plugins/H5Zbzip2.c

```diff
@@ -7,6 +7,8 @@
 /* Older versions of the hdf library may define H5PL_type_t here */
 #include <H5PLextern.h>
 
+#include "config.h"
```

Comment:
You need to put config.h before any other includes. Some of the settings in config.h may well impact other includes, so config.h has to come first.